### PR TITLE
feat: Input + renamed a color variable

### PR DESCRIPTION
**Pull request by manish-GH**
Preview: https://xenodochial-swirles-900c5f.netlify.app/components/input/

### CODE REVIEW

File: components/Input/styles.css [hunk 1]
```diff
@@ -1 +1,44 @@
...
+	box-shadow: 2px 2px 5px 0px var(--theme-card-box-shadow);
+	border-radius: 8px;
+	max-width: 25rem;
+	padding: 0 2rem;
+}
+
+.form .input {
+	border: 1px solid var(--theme-error);
+	outline: 1px solid var(--theme-error);
+}
```
Comment (vanshitamoolchandani) [inline components/Input/styles.css:36]: If you are giving same values to both border and outline properties, Why not give just a border of 2px. Other than that you are good to merge.

File: components/Input/index.html [hunk 2]
```diff
@@ -78,10 +78,78 @@ <h3 class="sidebar-head">Components</h3>
...
+						passwords. Use the class
+						<span class="highlight">input</span> for input component. You can
+						refer to the code given below.
+					</p>
+
+					<h2 class="component-sub-header">Input types</h2>
+
+					<div class="component-display vertical">
+						<input type="text" class="input" placeholder="Enter Text..." />
+						<input type="number" class="input" placeholder="Enter Number..." />
```
Comment (sri-rishi) [inline components/Input/index.html:95]: you should use input with label tag

**Review by sri-rishi — CHANGES_REQUESTED** [review]

File: components/Input/index.html (hunk 2 above)
Comment (manish-GH) [reply components/Input/index.html:95]: I assumed not all input fields require a label. I'll add this and more input types later on

File: components/Input/styles.css (hunk 1 above)
Comment (manish-GH) [reply components/Input/styles.css:36]: well this thickness was ideal. When I used 2px on border it was kind of thin and 2px on outline was kind of too thick. So I went with this

**Pull request merged**